### PR TITLE
Test on Node.js v0.12.x, v4.x, and v5.x

### PR DESCRIPTION
**Pull request by ntwb**

### CODE REVIEW

**Comment by magsout** [comment]
whats does it mean ? That travis tests with all this node version ?

**Comment by ntwb** [comment]
Yes, it tests with all the current release branches in the 0.12.x, 4.x.x, and 5.x.x branches.

The `iojs` branch is now deprecated now that the IOJS fork has merged back into NodeJS 4.x

**Comment by magsout** [comment]
Seems fair. thanks :+1:

**Comment by ntwb** [comment]
:+1: (Also brings it inline with the other stylelint repos in what versions of NodeJS tests run on Travis CI)

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,9 @@ sudo: false
 
 language: node_js
 node_js:
-  - iojs
+  - '5'
```
Comment (MoOx) [inline .travis.yml:5]: "stable" instead of 5 maybe ?

Comment (ntwb) [inline .travis.yml:5]: No, `stable` uses NodeJS v5.2.0 rather than the current v5.3.0

Comment (ntwb) [inline .travis.yml:5]: Further discussion on this and the pull requests I've made in the past couple of days:
• https://github.com/postcss/autoprefixer/pull/598
• https://github.com/sindresorhus/ava/pull/388

**Pull request merged**